src/runtime/gecko/localization.js

```diff
@@ -0,0 +1,57 @@
+import { Localization, contexts, translateDocument }
+  from '../../bindings/html';
+import { valueFromContext } from '../../lib/format';
+
+// Services.jsm is imported in entry points: l20n-html and l20n-xul
```

Comment:
We need `Services` for `Service.obs.addObserver` in the entry points.  Can I just import it twice?  It's memoized anyways, right?